src/core/config/OperationConfig.js

```diff
@@ -886,21 +886,43 @@ const OperationConfig = {
             }
         ]
     },
-    "Text encoding": {
-        description: "Translates the data between different character encodings.<br><br>Supported charsets are:<ul><li>UTF8</li><li>UTF16</li><li>UTF16LE (little-endian)</li><li>UTF16BE (big-endian)</li><li>Hex</li><li>Base64</li><li>Latin1 (ISO-8859-1)</li><li>Windows-1251</li></ul>",
-        run: CharEnc.run,
+    "Encode text": {
+        description: [
+            "Encodes text into the chosen character encoding.",
+            "<br><br>",
+            "Supported charsets are:",
+            "<ul>",
+            Object.keys(CharEnc.IO_FORMAT).map(e => `<li>${e}</li>`).join("<br>"),
```

Comment:
Do you mean:
1) `UTF-8 (65001)`
or
2) `65001 UTF-8`

With 1) you can quickly jump by typing `UTF` or `IBM`, with 2) you can jump by typing the codepage number.

I do agree that including the codepage number is a good idea as some names are very ambiguous, however I am leaning towards 1) rather than 2).